src/views/partials/breach-detail.js

```diff
@@ -129,7 +130,7 @@ function makeBreachDetails (breach) {
 
 export const breachDetails = data => `
   <header class="breach-detail-header">
-    <img class="breach-detail-logo breach-logo" alt="" src="https://monitor.cdn.mozilla.net/img/logos/${data.breach.LogoPath}" />
+    ${getBreachLogo(data.breach, data.breachLogos)}
```

Comment:
Doesn't it feel great to reuse your recently refactored module/util? 😊

Reply:
nice! :D